package.json

```diff
@@ -19,16 +19,16 @@
     "npm": ">=7.20.x <8.14"
   },
   "dependencies": {
-    "@bigcommerce/big-design": "^0.32.1",
+    "@bigcommerce/big-design": "^0.33.0",
```

Comment:
qq: does this break anything if we use the latest version of BD? `0.34.2`

Reply:
Nope! I'll push that up. 👍